pkg/kubelet/config/flags.go

```diff
@@ -52,7 +52,6 @@ type ContainerRuntimeOptions struct {
 func (s *ContainerRuntimeOptions) AddFlags(fs *pflag.FlagSet) {
 	// General settings.
 	fs.StringVar(&s.ContainerRuntime, "container-runtime", s.ContainerRuntime, "The container runtime to use. Possible value: 'remote'.")
```

Comment:
This should be removed along with `ContainerRuntime` related code.

Reply:
done